src/test/java/com/teamrocket/unit/clients/CustomerClientTest.java

```diff
@@ -0,0 +1,66 @@
+package com.teamrocket.unit.clients;
+
+import com.teamrocket.clients.CustomerClient;
+import com.teamrocket.model.CustomerDeliveryData;
+import com.teamrocket.proto.CustomerServiceGrpc;
+import com.teamrocket.proto.DeliveryData;
+import com.teamrocket.proto.SystemOrderId;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.TestPropertySource;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.reset;
+import static org.mockito.Mockito.when;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.NONE)
+@TestPropertySource(locations = {"classpath:applicationtest.properties"})
```

Comment:
I think your applicationtest.properties should be renamed to application-test.properties, and then use @ActiveProfiles("test") instead.
Or is this doing something else?

Reply:
It shouldn't event be here :D